akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -770,4 +770,26 @@ public void illustrateSourceQueue() throws Exception {
       }
     };
   }
+
+  @Test
+  public void illustrateSourceActorRef() throws Exception {
+    new TestKit(system) {
+      {
+        // #source-actorRef
+        int bufferSize = 10;
+
+        Source<Integer, ActorRef> source =
+            Source.actorRef(
+                bufferSize, OverflowStrategy.dropHead()); // note: backpressure is not supported
+        ActorRef actorRef =
+            source.map(x -> x * x).to(Sink.foreach(x -> System.out.println("got: " + x))).run(mat);
```

Comment:
I think this becomes more readable when putting 'map', 'to' and 'run' on separate lines. You could also use `runWith` rather than combining `to` and `run`.

Reply:
Tried to put them on separate lines, but the built in formatting puts all back on one line again :(

Reply:
Ah sorry :/